.github/workflows/pre-release.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     strategy:
       matrix:
         os: ['ubuntu-latest']
-        python-version: [3.9, '3.10', '3.11']
+        python-version: ['3.10', '3.11']
```

Comment:
I'm not super opposed to this, given that this app is run by our provided docker image "a-very-high-%" of the time. But what is the reason for exclusion? If its lack of active development, then 3.10 has stopped as well..

Reply:
The reason is just that we started depending on Python 3.10 syntax for type annotations, and rather than go back and rewrite a bunch, I just turned off testing <3.10 in the short term. With 3.12 out and us publishing an `env.yml` to get people up and running, I'm having a hard time mustering the desire to add it back without a push from someone else.

Reply:
That's fine by me - I'm definitely also in favor of limiting our dependency resolution matrix